src/client/components/pages/entity-revisions.js

```diff
@@ -54,6 +70,14 @@ class EntityRevisions extends React.Component {
 		);
 	}
 
+/**
+* Renders the data related to Revision such as 'author' and 'date'.
+* It also displays 'Revision Notes' column which displays the first revison note
+* which is a summary of the changes made in the revision.
+* @param {object} revision object is passed for the revison to be represented
```

Comment:
No need for "object is passed" - that's implied by `revision` being a parameter for the function.